index.html

```diff
@@ -0,0 +1,92 @@
+            </div>
+        </section>
+        <section class="card-section">
+            <div class="card">
+                <img class="card__img" src="images/main/Img_home_01.png">
+                <div class="card__text">
+                    <p class="card__category">Hot item</p>
+                    <h1 class="card__title">인기 상품을 <span class="text-wrap"> 확인해 보세요</span></h1>
+                    <p class="card__description">가장 HOT한 중고거래 물품을<br> 판다 마켓에서 확인해 보세요</p>
+                </div>
+            </div>
+        </section>
+        <section class="card-section">
+            <div class="card card--reverse">
+                <img class="card__img" src="images/main/Img_home_02.png">
+                <div class="card__text card__text--reverse">
+                    <p class="card__category">Search</p>
+                    <h1 class="card__title">구매를 원하는 <span class="text-wrap">상품을 검색하세요</span></h1>
+                    <p class="card__description">구매하고 싶은 물품은 검색해서<br> 쉽게 찾아보세요</p>
+                </div>
+            </div>
+        </section>
+        <section class="card-section">
+            <div class="card">
+                <img class="card__img" src="images/main/Img_home_03.png">
+                <div class="card__text">
+                    <p class="card__category">Register</p>
+                    <h1 class="card__title">판매를 원하는 <span class="text-wrap">상품을 등록하세요</span></h1>
+                    <p class="card__description">어떤 물건이든 판매하고 싶은 상품을<br> 쉽게 등록하세요</p>
+                </div>
+            </div>
+        </section>
+        <section class="banner-section bottom-banner">
+            <div class="banner">
+                <div class="banner__title banner__title--center">
+                    <h1 class="banner__text">믿을 수 있는 <span class="text-wrap">판다마켓 중고 거래</span></h1>
+                </div>
+                <img class="banner__img" src="images/main/Img_home_bottom.png">
+            </div>
+        </section>
+    </main>
+    <footer class="footer">
+        <span class="footer__name">©codeit - 2024</span>
+        <div class="footer-links">
+            <a class="footer__link" href="privacy.html">Privacy Policy</a>
+            <a class="footer__link" href="faq.html">FAQ</a>
+        </div>
+        <div class="footer-icons">
+            <a class="footer__icon" href="https://www.facebook.com" target="_blank">
+                <img src="images/icon/ic_facebook.png" alt="facebook">
+            </a>
+            <a class="footer__icon" href="https://x.com" target="_blank">
+                <img src="images/icon/ic_twitter.png" alt="twitter">
+            </a>
+            <a class="footer__icon" href="https://youtube.com" target="_blank">
+                <img src="images/icon/ic_youtube.png" alt="youtube">
+            </a>
+            <a class="footer__icon" href="https://instagram.com" target="_blank">
+                <img src="images/icon/ic_instagram.png" alt="instagram">
+            </a>
+        </div>
```

Comment:
💊 제안
이 영역은 로고들의 모음이라고 할 수 있습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
        <ul class="footer-icons">
            <li>
              <a class="footer__icon" href="https://www.facebook.com" target="_blank">
                  <img src="images/icon/ic_facebook.png" alt="facebook">
              </a>
            </li>
        </ul>
```